common/anomalies/hfe_anomaly.txt

```diff
@@ -333,26 +343,27 @@ hfe_anomaly_future_advisor = {
 	max_once_global = yes
 	null_spawn_chance = @null_spawn_chance
 	spawn_chance = {
+		base = 0
 		modifier = {
 			add = 5
 			hfe_trigger_story_enabled = yes
 			is_planet_class = pc_gas_giant
 			has_ring = yes
 		}
 		modifier = {
-			factor = 2
+			factor = 10
 			years_passed > 30
 		}
 		modifier = {
-			factor = 2
+			factor = 20
 			years_passed > 60
 		}
 		modifier = {
-			factor = 2
+			factor = 20
 			years_passed > 90
 		}
 		modifier = {
-			factor = 2
+			factor = 30
```

Comment:
This seems too huge

Reply:
Okay, I lowered it to match the ninety year interval